eng/scripts/Save-Package-Namespaces-Property.ps1

```diff
@@ -15,26 +15,59 @@ PackageInfo subdirectory. The whl files are going to be in the subdirectory whic
 is the same as the artifact's name but artifact name in the file's name will have
  underscores instead of dashes.
 
-.PARAMETER ArtifactsList
-The list of artifacts to gather namespaces for, this is only done for libraries that are
-producing docs.
--ArtifactsList ('${{ convertToJson(parameters.Artifacts) }}' | ConvertFrom-Json)
+.PARAMETER RepoRoot
+The root of the directory. This will be used in combination with the paths of of
+the properties within each PackageInfo file to determine the full path to other relevant
+metadata...like ci.yml for instance.
 #>
 [CmdletBinding()]
 Param (
     [Parameter(Mandatory = $True)]
     [string] $ArtifactStagingDirectory,
-    [Parameter(Mandatory=$true)]
-    [array] $ArtifactsList
+    [Parameter(Mandatory = $True)]
+    [string] $RepoRoot
```

Comment:
`common.ps1` already exports a variable $RepoRoot

Reply:
Added before I added the import of common.ps1. Will cleanup.